Headers/DebugServer2/GDBRemote/DummySessionDelegateImpl.h

```diff
@@ -99,6 +99,7 @@ class DummySessionDelegateImpl : public SessionDelegate {
   ErrorCode onTerminate(Session &session, ProcessThreadId const &ptid,
                         StopInfo &stop) override;
   ErrorCode onTerminate(Session &session, ProcessId pid) override;
+  void onExit(Session &session) override;
```

Comment:
Can you name `onExit` something more explicit like `onExitServer` or `onTerminateServer`. It's not 100% clear from the name that we're talking about exiting the server, and not the inferior here.